v_0_1_0/master_template.go

```diff
@@ -117,7 +117,18 @@ write_files:
           namespace: kube-system
           labels:
             k8s-app: calico-node-controller
+          annotations:
+            scheduler.alpha.kubernetes.io/critical-pod: ''
         spec:
+          tolerations:
+          - key: node-role.kubernetes.io/master
+            operator: Exists
+            effect: NoSchedule
+          - key: CriticalAddonsOnly
+            operator: Exists
+          nodeSelector:
+            role: master
```

Comment:
Please use label that used in community (e.g. calico, kubeadm, etc.)

https://github.com/giantswarm/etcd-backup/blob/master/helm/etcd-backup-chart/templates/cronjob.yaml#L28-L33